Getting Started with Form (#1485)
DevExpress/devextreme-documentation: concepts/05 UI Components/Form/01 Getting Started with Form/40 Organize the Form Items/40 Add an Empty Space.md

> If you need to add an empty space between neighboring items, use an empty item. To create it, assign "empty" to the [itemType](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/EmptyItem/#itemType) property. To define how many columns the empty item must span, specify the [colSpan](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/EmptyItem/#colSpan) option. For the full list of available properties, visit the [Empty Item](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/EmptyItem/) section.

Reviewer: ```suggestion
If you need to add an empty space between neighboring items, use an empty item. To create an empty item, assign "empty" to the [itemType](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/EmptyItem/#itemType) property. To define how many columns the empty item should span, specify the [colSpan](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/EmptyItem/#colSpan) option. For a list of available properties, refer to the [Empty Item](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/EmptyItem/) section.
```

> In the following example, the empty item [spans](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/EmptyItem/#colSpan) two columns. The resulting layout looks as follows:

Reviewer: Why do you add "The resulting layout looks as follows"? You've already explained that there is an example and what this example is about.

Author: This sentence introduces the following image:

![form-getting-started-empty-item](https://user-images.githubusercontent.com/33040934/106897508-12ed3e00-6704-11eb-8a1a-f05909d7115a.png)
